docs/tutorials/start-a-private-network/4customchain.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Creating Your Private Network
+---
+Now that each participant has their own keys generated, you're ready to start your own custom chain.
+
+> Validators should not share the same keys, even for learning purposes. If two validators have the same keys, they will produce conflicting blocks, and be slashed.
```

Comment:
```suggestion
> Validators should not share the same keys, even for learning purposes. If two validators have the same keys, they will produce conflicting blocks and be slashed.
```

Reply:
I believe it is correct as written. https://www.grammarly.com/blog/comma-before-and/ has the following example 
![image](https://user-images.githubusercontent.com/2915325/69094425-533c9f80-0a1e-11ea-96ca-0297293cdf25.png)

Simplifying my sentence it becomes:
Incorrect: They will produce blocks, and be slashed.
  Correct: They will produce block and be slashed.

Even though nobody asked: Although it appears syntactically similar, this is not an example of the [Oxford comma](https://en.wikipedia.org/wiki/Serial_comma) of which I'm a big fan.

Reply:
I am suggesting to remove the comma, which you seem to agree is correct?

Reply:
Guess I forgot how to read diffs there for a sec.